.github/workflows/ci.yml

```diff
@@ -11,8 +11,8 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-node@v1
+    - uses: Brightspace/third-party-actions@actions/checkout
+    - uses: Brightspace/third-party-actions@actions/setup-node
```

Comment:
This will become mandatory once we flip a switch in the org (no ETA).
Why do we do this?

* Pinning: when you use `@v2` you get updates out of band. In Brightspace/third-party-actions we mirror/fork the repo and use dependabot to get updates based on a tag... so we are pinned at the org level (but your repo is floating, just the same)
* License compliance: we have a central place to audit that
* Security etc.
* A choke point to apply "good taste", e.g. force us to standardize on one solution to a particular problem, where that makes sense

We've found that it's actually quite easy to just review all diffs as they come in from third-party things if we use dependabot. It probably makes sense to do this once for the company though.

Reply:
I'll write something like this down in the docs for third-party-actions at some point when i get time to polish it

Reply:
In the meantime, is there a way to discover which actions are available as forks?

Reply:
Yep, there is a list in the README for third-party-actions: https://github.com/brightspace/third-party-actions/tree/master/README.md